packages/sku/entry/render/index.js

```diff
@@ -37,6 +39,7 @@ export default async (renderParams) => {
       ...renderContext,
       _addChunk: (chunkName) => extractor.addChunk(chunkName),
       SkuProvider,
+      renderToString: (node) => renderToString(node),
```

Comment:
Great comment. I thought about leaving an inline comment on this, maybe I should.

I deliberately added this to avoid exposing the options to the user. Exposing configuration is only done after extensive scrutiny, as much as possible we opt for conformity over configuration.

I still wanted to add the second parameter to make the arbitrary timeout figure I set visible and make it easy to configure (internally) in the future.

Reply:
Changes to a CONST anyway to avoid confusion / having people think it's configurable by the consumer.